### ISSUE TITLE
Some candidate datasets

### ISSUE THREAD

**Issue by nheeren** [nheeren/material_intensity_db#1]
The following datasets contains material intensity data that may be included into the database.

*Volunteers are encouraged to extract the data and to create a pull request.*

- Blengini and di Carlo 2010 "The changing role of life cycle phases, subsystems and materials in the LCA of low energy buildings" https://www.sciencedirect.com/science/article/pii/S0378778810000022
- Beccali et al 2013 "Energy retrofit of a single-family house: Life cycle net energy saving and environmental benefits" https://www.sciencedirect.com/science/article/pii/S1364032113003432
- Dodoo et al 2010 "Life cycle primary energy implication of retrofitting a wood-framed apartment building to passive house standard" https://www.sciencedirect.com/science/article/pii/S0921344910000856
- Gaspar and Santos 2015 "Embodied energy on refurbishment vs. demolition: A southern Europe case study" https://www.sciencedirect.com/science/article/pii/S0378778814009712
- Gontia etl al 2018 "Material-intensity database of residential buildings: A case-study of Sweden in the international context" https://www.sciencedirect.com/science/article/pii/S0921344917304135

**Comment by TomerFishman** [comment]
Residential building material stocks and component-level circularity: The case of Singapore, JCP 2019,
Mohit Arora, Felix Raspall, Lynette Cheah Arlindo Silva https://doi.org/10.1016/j.jclepro.2019.01.199
-I couldn't find the actual MI values in the manuscript or SI, so we may have to contact the authors.

**Comment by nheeren** [comment]
Shirazi and Ashuri 2018, Embodied life cycle assessment comparison of single family residential houses considering the 1970s transition in construction industry: Atlanta case study, https://doi.org/10.1016/j.buildenv.2018.05.021

**Comment by nheeren** [comment]
Keoleian et al., Journal of Industrial Ecology, 2000, https://doi.org/10.1162/108819800569726

Thanks @peterberr

**Comment by anistad** [comment]
- Yang et al. (2020) Urban buildings material intensity in China from 1949 to 2015 https://doi.org/10.1016/j.resconrec.2020.104824. MI of 813 buildings in China available here: https://zenodo.org/record/2704917#.XykdOPkzaUk 

- Mao et al. (2020) High-Resolution Mapping of the Urban Built Environment Stocks in Beijing https://doi.org/10.1021/acs.est.9b07229 Table S6c in SI.

**Comment by TomerFishman** [comment]
Nice to see some activity here!

Schandl et al 2020 A spatiotemporal urban metabolism model for the Canberra suburb of Braddon in Australia http://www.sciencedirect.com/science/article/pii/S0959652620318175 (table 6)

**Comment by peterberr** [comment]
Another one, looks very relevant if we haven't got it yet: Looks like a competing/complementary effort to the Heeren and Fishman database?
Marinova et al. (2020) Global construction materials database and stock analysis of residential buildings between 1970-2050
https://doi.org/10.1016/j.jclepro.2019.119146

**Comment by TomerFishman** [comment]
Uruguay: https://doi.org/10.1016/j.jclepro.2020.121958 may need conversion from total mass to per-sq meter
South Korea: https://doi.org/10.1016/j.jclepro.2020.120385
Beijing: https://pubs.acs.org/action/showCitFormats?doi=10.1021/acs.est.9b07229&ref=pdf
Fiji: http://dx.doi.org/10.3390/su12030834
London: https://doi.org/10.1016/j.rcrx.2019.100027

**Comment by nheeren** [comment]
@mroeck will soon publish a buildings dataset that can be integrated here eventually.

**Comment by peterberr** [comment]
I also had a [publication](http://doi.org/10.5334/bc.126) last year which contains material intensities for many US residential archetypes that could be added. 
Should I add in these and then make a pull request?

**Comment by TomerFishman** [comment]
Nice to see some activity here! :-)
As I updated Niko a month or two ago, through my collaboration with IIASA we now have over 800 datapoints and a lot of analysis, which we'll update here soon(ish)

**Comment by nheeren** [comment]
> Nice to see some activity here! :-) As I updated Niko a month or two ago, through my collaboration with IIASA we now have over 800 datapoints and a lot of analysis, which we'll update here soon(ish)

You mentioned that you will be using a new structure. Possibly it would make sense to adopt that already now so data contributions that come now don't need to be revised immediately.

**Comment by peterberr** [comment]
> > Nice to see some activity here! :-) As I updated Niko a month or two ago, through my collaboration with IIASA we now have over 800 datapoints and a lot of analysis, which we'll update here soon(ish)
> 
> You mentioned that you will be using a new structure. Possibly it would make sense to adopt that already now so data contributions that come now don't need to be revised immediately.

Yes, It would be nice to see the structure before I add some new data

**Comment by mroeck** [comment]
> @mroeck will soon publish a buildings dataset that can be integrated here eventually. 

Thanks for noting this already. Looking forward to add our data soon-ish!

**Comment by TomerFishman** [comment]
Here's the structure with a small data sample in Excel. The new columns are BW-CE, with standard options to choose from.
https://www.dropbox.com/s/dpun2g944ukc3fm/buildings_v2_sample.xlsx?dl=0

@peterberr one of my students already entered data from your study, so maybe you'd like to review it first

**Comment by peterberr** [comment]
> Here's the structure with a small data sample in Excel. The new columns are BW-CE, with standard options to choose from. https://www.dropbox.com/s/dpun2g944ukc3fm/buildings_v2_sample.xlsx?dl=0
> 
> @peterberr one of my students already entered data from your study, so maybe you'd like to review it first

Thanks for the structure. Sure, nice that the data from my study was already entered! Would be nice to review. Is there a working version of the database or somewhere I can see that? The current data file in this repository hasn't been updated in some years.

**Comment by TomerFishman** [comment]
> Thanks for the structure. Sure, nice that the data from my study was already entered! Would be nice to review. Is there a working version of the database or somewhere I can see that? The current data file in this repository hasn't been updated in some years.

Check the dropbox file again, it's there now :-)
